database/test/conftest.py

```diff
@@ -27,6 +27,24 @@
 codes_count: int = 16  # adjust me when adding tables
 matview_count: int = 0  # adjust me when adding views
 
+
+class IndexMeta(NamedTuple):
+    columns: Tuple[str, ...]
+    unique: bool
+
+
+ordering_index_by_table = {
```

Comment:
@Rikuoja, I don't quite like defining the expected indexes manually like this. Do you have ideas how to avoid this?